src/main/java/net/mcreator/element/types/MusicDisc.java

```diff
@@ -59,6 +60,8 @@ public class MusicDisc extends GeneratableElement implements IItem, IItemWithTex
 
 	public MusicDisc(ModElement element) {
 		super(element);
+
+		this.rarity = "RARE";

```

Comment:
By doing this, you don't need converter

Reply:
But it doesn't work, the rarity that I find on the music disc is common

Reply:
I see. This is because this is implemented incorrectly. 

Check other mod elements how this is done and read a bit on GSON deserialization and why default constructor is needed when specifying "default values" for GSON.

This can be a starting point: https://stackoverflow.com/questions/18645050/is-default-no-args-constructor-mandatory-for-gson

Reply:
I looked at the Item element code to do the rarity

Reply:
> read a bit on GSON deserialization and why default constructor is needed when specifying "default values" for GSON.

> This can be a starting point: https://stackoverflow.com/questions/18645050/is-default-no-args-constructor-mandatory-for-gson

Reply:
![image](https://github.com/MCreator/MCreator/assets/16374228/ee0d3044-e21f-4466-b64d-b8588fc5a7cd)

Reply:
But please read https://stackoverflow.com/questions/18645050/is-default-no-args-constructor-mandatory-for-gson so you understand why this is needed

Reply:
Ok, thank you